.github/workflows/test.yml

```diff
@@ -0,0 +1,62 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches: 
+     - master
+
+env:
+  GO111MODULE: on
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # The "build" workflow
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    
+    # Setup Go
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.11' # The Go version to download (if necessary) and use.
```

Comment:
Lets use tip if possible

Reply:
I've used [this ](https://github.com/maxatome/install-go) to install tip, there is no easy way to get tip version directly.
https://github.com/actions/setup-go/issues/21